pkg/manifests/yaml/rte/daemonset.yaml

```diff
@@ -14,11 +14,10 @@ spec:
       serviceAccountName: rte
       containers:
       - name: resource-topology-exporter-container
-        image: ${RTE_CONTAINER_IMAGE}
+        image: quay.io/k8stopologyawareschedwg/resource-topology-exporter:latest
         command:
         - /bin/resource-topology-exporter
-        - --export-namespace=${EXPORT_NAMESPACE}
```

Comment:
What about ```--export-namespace```? I guess that now it will be the ```default``` ns, but this is bad, especially in tests which will be looking for the NRT object on a specific ns (usually the one the tests components` are running in)

Reply:
good point, I should have been more explicit. Once https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/74 is merged, we will need no export namespace anymore. Actually the option will be gone entirely.
This means this PR should be merged only after https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/74 is merged.